ddp-workspace/projects/ddp-dsm-ui/src/app/shipping-search/shipping-search.component.ts

```diff
@@ -121,43 +120,39 @@ export class ShippingSearchComponent implements OnInit {
       ); // need to subscribe, otherwise it will not send!
   }
 
-  valueChanged( value: any, parameterName: string, kitRequest: KitRequest ): void {
-    let v;
-
-    if (typeof value === 'string') {
-      kitRequest[ parameterName ] = value;
-      v = value;
-    }
-    if (v != null) {
-      const realm: string = kitRequest.realm;
-      const patch1 = new PatchUtil(
-        kitRequest.dsmKitRequestId, this.role.userMail(),
-        {
-          name: parameterName,
-          value: v
-        }, null, 'dsmKitRequestId', kitRequest.dsmKitRequestId,
-        'kit', null, realm, kitRequest.participantId
-      );
-      const patch = patch1.getPatch();
-      this.currentPatchField = parameterName;
-      this.patch( patch );
-    }
+  patch(patch: any): Observable<any> {
+    return this.dsmService.patchParticipantRecord(JSON.stringify(patch));
   }
 
-  patch( patch: any ): void {
-    this.dsmService.patchParticipantRecord( JSON.stringify( patch ) ).subscribe( { // need to subscribe, otherwise it will not send!
-      next: data => {
-        this.currentPatchField = null;
-      },
-      error: err => {
-        if (err._body === Auth.AUTHENTICATION_ERROR) {
-          this.auth.logout();
-        }
-      }
-    } );
+//Triggers when user finishes typing
```

Comment:
Is this comment still valid?

Reply:
Yes, I believe that is still valid. The valueChanged function is still called on dateChanged, which is called when the user finishes typing.